services/pull/merge.go

```diff
@@ -133,6 +136,11 @@ func expandDefaultMergeMessage(template string, vars map[string]string) (message
 	return os.Expand(message, mapping), os.Expand(body, mapping)
 }
 
+// GetDefaultMergeMessage returns default message used when merging pull request
+func GetDefaultMergeMessage(ctx context.Context, baseGitRepo *git.Repository, pr *issues_model.PullRequest, mergeStyle repo_model.MergeStyle) (message, body string, err error) {
+	return getMergeMessage(ctx, baseGitRepo, pr, mergeStyle, map[string]string{})
```

Comment:
```suggestion
	return getMergeMessage(ctx, baseGitRepo, pr, mergeStyle, nil)
```

it could be `nil` here, Golang is safe to do `for range` on a nil map.